packages/gafl-webapp-service/src/pages/licence-details/licence-length/__tests__/result-function.spec.js

```diff
@@ -0,0 +1,54 @@
+import resultFunction from '../result-function'
+import { CommonResults } from '../../../../constants.js'
+import { licenceToStart } from '../../licence-to-start/update-transaction.js'
+
+describe('licence-length > result-function', () => {
+  const mockStatusCacheGet = jest.fn(() => ({}))
+  const mockTransactionCacheGet = jest.fn(() => ({}))
+
+  const mockRequest = {
+    cache: () => ({
+      helpers: {
+        status: {
+          getCurrentPermission: mockStatusCacheGet
+        },
+        transaction: {
+          getCurrentPermission: mockTransactionCacheGet
+        }
+      }
+    })
+  }
+
+  describe('default', () => {
+    beforeEach(jest.clearAllMocks)
+
+    it('should return SUMMARY if status is from summary', async () => {
+      mockStatusCacheGet.mockImplementationOnce(() => ({
+        fromSummary: true
+      }))
+      const result = await resultFunction(mockRequest)
+      expect(result).toBe(CommonResults.SUMMARY)
+    })
+
+    it('should return OK if status is not from summary and licenceToStart is after payment', async () => {
+      mockTransactionCacheGet.mockImplementationOnce(() => ({
+        licenceToStart: licenceToStart.AFTER_PAYMENT
+      }))
+      const result = await resultFunction(mockRequest)
+      expect(result).toBe(CommonResults.OK)
+    })
+
+    it('should return OK if status is not from summary and licenceLength is 12 months', async () => {
```

Comment:
With this and the preceding test, is it worth explicitly setting `fromSummary: false` to match the test description?